Classes/EventListener/EventListenerManager.php

```diff
@@ -24,7 +28,7 @@
  *
  * @Flow\Scope("singleton")
  */
-class EventListenerLocator
+class EventListenerManager
```

Comment:
I agree, the naming isn't optimal and also agree that it breaks the single-responsibility-principle. 

I'm not sure about `EventListenerConsistencyManager` though. How about moving `catchUp` to the `EventPublisher`. That's much closer to the topic.